src/views/Beheer/contactverzoek-formulieren/ContactverzoekFormulierBeheer.vue

```diff
@@ -38,7 +37,7 @@
     </label>
     <!-- dropdown for afdelingen -->
     <label class="utrecht-form-label">
-      <span>Afdeling *</span>
+      <span>Afdeling <span class="required-label">*</span></span>
```

Comment:
dit wijkt af van bijvoorbeeld het contactverzoekformulier. Een class="required" zou moeten volstaan denk ik.
de custom required-label css in dit component kan dan weg.

Reply:
ja ik heb deze aanpassing verwijderd. Na overleg hebben we ook besloten dat we geen maatwerk styling zullen toepassen specifiek voor deze pagina. In plaats daarvan een algemene styling voor alle beheerpagina’s om consistentie te waarborgen.